### PR TITLE
[CVP-1753] Remove support for encrypted kube objects with CVP ssh key

### PR DESCRIPTION
**Pull request by sonam1412**
This is to remove the support for the old RSA encryption method of the kube objects

### CODE REVIEW

**Review by jsztuka — APPROVED** [review]
lgtm

**Review by dirgim — APPROVED** [review]
LGTM

**Review by mvalarh — APPROVED** [review]

**Review by nmars — APPROVED** [review]
LGTM

File: roles/inject_openshift_kube_objects/tasks/main.yml [hunk 1]
```diff
@@ -5,32 +5,6 @@
...
-
-    - name: "Read the encrypted kube_objects decode and store them in a file"
-      shell: "echo \"{{ kube_objects }}\" | tr \" \" \"\n\" | base64 --decode > {{ kube_objects_dir }}/get_kubeObjects.txt"
-      no_log: true
-
-    - name: "Decrypt the kube_objects through the shared symmetric key"
-      shell: "openssl aes-256-cbc -d -in {{ kube_objects_dir }}/get_kubeObjects.txt -out {{ kube_objects_dir }}/kube_objects.yaml -pass file:{{ kube_objects_dir }}/secret.key"
-
 - name: "Decrypt the kube_objects if it's PGP encoded."
   when: "'-----BEGIN PGP MESSAGE-----' in kube_objects"
```
Comment (lslebodn) [inline roles/inject_openshift_kube_objects/tasks/main.yml:9]: Do we expect that `-----BEGIN PGP MESSAGE-----` will not be in the `kube_objects`

Can we remove unnecessary block ?

**Review by lslebodn — CHANGES_REQUESTED** [review]
Please rebase on top of current master

File: roles/inject_openshift_kube_objects/tasks/main.yml (hunk 1 above)
Comment (sonam1412) [reply roles/inject_openshift_kube_objects/tasks/main.yml]: Done

**Comment by sonam1412** [comment]
@lslebodn please have another look

**Review by lslebodn — APPROVED** [review]
LGTM

**Pull request merged**